packages/mdc-feature-targeting/_mixins.scss

```diff
@@ -0,0 +1,12 @@
+@import "./functions";
+
+// Mixin that annotates the contained styles as applying to specific cross-cutting features
+// indicated by the given list of feature targets.
+@mixin mdc-feature-targets($feature-targets...) {
+  $valid: mdc-feature-verify-target_($feature-targets...);
```

Comment:
Nice catch. Yeah, it'll catch it in parse anyway, there'll just be one more line in the stack which I don't think is a big deal since there's 4-5 of them anyway.